backend/placeholder/api/models.py

```diff
@@ -78,10 +74,10 @@ class Photo(models.Model):
     Every user can add photos to his albums.
     """
 
-    title           = models.CharField(max_length=255)
-    url             = models.CharField(max_length=255)
-    thumbnail_url   = models.CharField(max_length=255)
-    album           = models.ForeignKey('Album', on_delete=models.CASCADE)
+    title           = models.CharField(max_length=255, null=True)
```

Comment:
Why can this be null? Also, a char field should be `blank=True` if you want it to be blank..